File: 프로그래머스/unrated/132267. 콜라 문제/콜라 문제.py
def solution(a, b, n):
    answer = 0
    while(n >= a): # a 병 미만일 때는 안준디~~
        remain = n % a # 나머지
        quo = n // a # 몫, 나머지 뺌
        answer += quo * b
        n = quo * b + remain
    
    return answer

File: 프로그래머스/1/132267. 콜라 문제/콜라 문제.java
class Solution {
    public int solution(int a, int b, int n) {
        int answer = 0;
        
        while(n >= a){
            int remain = n % a;
            int quo = n / a;
            answer += quo * b;
            n = quo * b + remain;
        }
        
        return answer;
    }
}